### ISSUE TITLE
"Add to Home Screen" feature doesn't work

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [gauntface/simple-push-demo#79]

**Comment by gauntface** [comment]
Sorry but this is a demo for push and I'm not expecting users to add it to their homescreen

**Comment by FluorescentHallucinogen** [comment]
> I'm not expecting users to add it to their homescreen
In this case, what manifest.json file is for? ;)

IMO, receiving push notifications from "homescreen apps" in background is very interesting use case.

**Comment by FluorescentHallucinogen** [comment]
Looks like it's very easy to fix, the problem is in https://github.com/gauntface/simple-push-demo/blob/master/src/manifest.json#L11

**Comment by FluorescentHallucinogen** [comment]
![1](https://cloud.githubusercontent.com/assets/7892779/17219433/17e36406-54f4-11e6-9505-38b6f4fd9d89.png)
![2](https://cloud.githubusercontent.com/assets/7892779/17219436/1af60d92-54f4-11e6-9263-850a3cfde85d.png)
![3](https://cloud.githubusercontent.com/assets/7892779/17219441/1dcd299c-54f4-11e6-89d6-fa4152d8ae46.png)
![4](https://cloud.githubusercontent.com/assets/7892779/17219460/22af0890-54f4-11e6-8dea-d18c9efca358.png)
![5](https://cloud.githubusercontent.com/assets/7892779/17219467/27a8d916-54f4-11e6-8e59-88dc17eee2ee.png)
![6](https://cloud.githubusercontent.com/assets/7892779/17219473/2b8add90-54f4-11e6-82bf-5a3c17d086d9.png)

**Comment by FluorescentHallucinogen** [comment]
If you fix "Add to Home Screen" feature, it will be possible to add your demo to https://pwa.rocks as real Progressive Web App. ;-)

**Comment by FluorescentHallucinogen** [comment]
Seems it's GitHub Pages bug. Just cloned `gh-pages` branch and deployed it "as is" to Firebase (https://firebase.google.com) and all work like a charm! I've visited the Firebase-deployed app at least twice, with at least five minutes between visits, and Chrome for Android automatically displays "Add to Home Screen" banner. I've added it to home screen, then clicked app icon on home screen, and the app works fine.
